pkg/kubelet/eviction/types.go

```diff
@@ -66,14 +66,23 @@ type Config struct {
 	Thresholds []Threshold
 }
 
+type ThresholdValue struct {
```

Comment:
nit: some godoc like // ThresholdValue is a value holder that abstracts literal versus percentage based quantity